src/pymovements/utils/archives.py

```diff
@@ -150,7 +151,10 @@ def _extract_tar(
         Compression filename suffix.
     """
     with tarfile.open(source_path, f'r:{compression[1:]}' if compression else 'r') as archive:
-        archive.extractall(destination_path)
+        if sys.version_info < (3, 12):  # pragma: <3.12 cover
+            archive.extractall(destination_path)
+        else:  # pragma: >=3.12 cover
```

Comment:
great that this pragma directive works. can you point me to a reference to get more information on this?